R/toxlsx.R

```diff
@@ -96,6 +82,29 @@ toxlsx <- function(object,
     output_name <- deparse(object)
   }
 
+  if (!is_list) tosheet <- if_atomic_to_list(tosheet, output_name)
```

Comment:
Actually `if (!is_list)` is not necessary because it could be wanted to recycle the tosheet argument in order to get all the dataframes in one sheet.

```suggestion
tosheet <- if_atomic_to_list(tosheet, output_name)
```